### DISCUSSION TITLE
How can I autostart a server when it's offline and someone tries to connect?

### DISCUSSION THREAD

**Discussion by ItsLiyua** [haveachin/infrared#166] [Q&A]
That's more or less my question. I have an infrared instance and a minecraft server. However my server is not starting when I try connecting to the infrared instance. They are both on the same device and in the same virtual network. The infrared instance has the container's name in the config and I'm passing the hosts `/var/run/docker.sock` to the infrared container. Any idea why my server is not starting when I try to connect to infrared?

**Comment by haveachin** [comment]
What version of Infrared are you using? Automatically starting/stopping a Docker container is currently only available in v1. If you are using v2 you could use [itzg/docker-minecraft-server Autopause](https://github.com/itzg/docker-minecraft-server#autopause) feature.

**Reply by ItsLiyua** [reply]
Alright thanks. I'll look into that. I was using the latest build. I basically just ran the commands from the readme about 6 hours ago.
